include/openssl/types.h

```diff
@@ -6,10 +6,22 @@
  * in the file LICENSE in the source distribution or at
  * https://www.openssl.org/source/license.html
  */
+ 
+/*
+ * Unfortunate workaround to avoid symbol conflict with wincrypt.h
+ * See https://github.com/openssl/openssl/issues/9981
+ */
+#ifdef _WIN32
+# define WINCRYPT_USE_SYMBOL_PREFIX
```

Comment:
Oh interesting. Is this something the Windows headers will later consume to emit different constants?

Reply:
Yes - that is the plan

Reply:
I've got this approved by folks on my team to update wincrypt.h to respect WINCRYPT_USE_SYMBOL_PREFIX in future.

Reply:
@samuel-lee-msft Do you have an estimate for when the updated wincrypt.h will be available?